reference/forms/types/entity.rst

```diff
@@ -202,9 +202,9 @@ query_builder
 Allows you to create a custom query for your choices. See
 :ref:`ref-form-entity-query-builder` for an example.
 
-The value of this option can either be a ``QueryBuilder`` object or a Closure.
+The value of this option can either be a ``QueryBuilder`` object, a Closure or null.
```

Comment:
unless I'm missing something, it's not the option value that can be `null`. Instead, the closure can return null.